app/navigation/Navigator.scala

```diff
@@ -41,7 +41,7 @@ class Navigator @Inject()() {
     case propertyrentals.ClaimPropertyIncomeAllowancePage => taxYear => _ => PropertyRentalsCheckYourAnswersController.onPageLoad(taxYear)
     case IsNonUKLandlordPage => taxYear => userAnswers => isNonUKLandlordNavigation(taxYear, userAnswers)
     case DeductingTaxPage => taxYear => _ => routes.IncomeFromPropertyRentalsController.onPageLoad(taxYear, NormalMode)
-    case IncomeFromPropertyRentalsPage => taxYear => _ => routes.TotalIncomeController.onPageLoad(taxYear, NormalMode) // TODO: route to CYA page
+    case IncomeFromPropertyRentalsPage => taxYear => _ => LeasePremiumPaymentController.onPageLoad(taxYear, NormalMode)
```

Comment:
A test should be either fixed or added in the `NavigatorSpec`?

Reply:
Done :+1: